src/c++/perf_analyzer/command_line_parser.cc

```diff
@@ -121,6 +122,8 @@ CLParser::Usage(const std::string& msg)
   std::cerr << "\t--sequence-id-range <start:end>" << std::endl;
   std::cerr << "\t--string-length <length>" << std::endl;
   std::cerr << "\t--string-data <string>" << std::endl;
+  std::cerr << "\t--json-input-data" << std::endl;
```

Comment:
I don't love these option names. It implies that the knob takes data instead of being a boolean switch.  Maybe add "enable" or some other boolean word into the name?

Reply:
Discussed offline. Now called `--input-content-type=[binary|json]` and `--output-content-type=[binary|json]` 👍